libimgui-platform-osx/imgui/buildfile

```diff
@@ -1,7 +1,7 @@
 intf_libs = # Interface dependencies.
 import intf_libs += libimgui%lib{imgui}
 
-platform_libs = -framework AppKit -framework ModelIO -framework GameController
+platform_libs = -framework AppKit -framework ModelIO -framework GameController -framework QuartzCore
```

Comment:
What's the reason that you had to add this flag in the non-docking package? I think it already used to work without it?

Reply:
I added it for consistency; in retrospect, this is indeed not necessary. I will remove. Thank!